cfi_forward.adoc

```diff
@@ -212,6 +212,9 @@ else
 endif
 ----
 
+Whereas `lpcll` is the only instruction that can execute when `ELP` is
```

Comment:
Is "Whereas" needed here?

Reply:
Yes, since it is stating that lpcll can execute in either state of ELP.